src/helpers/getByAPI.js

```diff
@@ -1,24 +1,44 @@
 // @flow
 import * as React from 'react';
+import prettyFormat from 'pretty-format';
 import ErrorWithStack from './errorWithStack';
 
+const getNodeByType = (node, type) => node.type === type;
+
 const getNodeByName = (node, name) =>
-  node.type.name === name ||
-  node.type.displayName === name ||
-  node.type === name;
+  typeof node.type !== 'string' &&
+  (node.type.displayName === name || node.type.name === name);
+
+const getNodeByText = (node, text) => {
+  try {
+    // eslint-disable-next-line
+    const { Text, TextInput } = require('react-native');
```

Comment:
As I understand, we make this `require` call here to make lib compatible with other environments?

Reply:
There may be different text primitives in different environments (like ReactVR). I stand that it's safer to assert on right primitives instead of using string-based comparisons, where the display name may change over time.